charts/common/tests/externalsecret_test.yaml

```diff
@@ -0,0 +1,156 @@
+values: &values
+  env: dev
+  app: testsuite
+  shortname: tstsut
+  team: common
+  labels:
+    custom: label
+  ingress:
+    host: test.dev.entur.io
+    trafficType: public
+  secrets:
+    one-secret:
+      - FIRST_SECRET
+    two-secrets:
+      - FIRST_SECRET
+      - SECOND_SECRET
```

Comment:
```suggestion
      - SECOND_SECRET
      - THIRD_SECRET
```

Reply:
Tests needs to be updated to reflect this disambiguation.

Reply:
Renamed the secret in `one-secret` to SINGLE_SECRET, and updated the tests to reflect the change.